### DISCUSSION TITLE
User Story Catch up

### DISCUSSION THREAD

**Discussion by zeuchste** [nebulastream/nebulastream#12] [Ideas]
@kisa7 maybe we could use this here to catch up with the great work you did on the user stories and what is missing from that end as this would be highly valuable for the first milestone already

**Comment by kisa7** [comment]
Thank you for bringing it up. We addressed most of the issues that people found. However, there are still some bugs coming from Dwi and Nils. We can either try to fix them or first create a stable worker and then redo testing for the user stories.

**Comment by kisa7** [comment]
The testing progress can be found here: https://docs.google.com/spreadsheets/d/1tRCP44eJohWT1b1BgdM3M81ZW3m0qgq_DEEMlW85fvQ/edit?usp=sharing

The user stories here: https://docs.google.com/document/d/1-SY4VAKvPmdx-49E5V_tzgp69GzwxJWe55kbqSL8YGM/edit?usp=sharing

**Reply by zeuchste** [reply]
Thanks for bringing this here, did we now do already all user stories once? If not we can also then create issues for that and give it to Hiwis. Also do we need to discuss new stories?

**Reply by zeuchste** [reply]
In addition, we thought of creating tests for each of them to make sure that they do not break in the future, did we do them?

**Reply by kisa7** [reply]
Well, I started creating tests for user stories but only for the configuration validation, which was my part
